### PR TITLE
Fix issues in app settings table

### PR DESCRIPTION
**Pull request by Flanker32**
- Fix app settings table may have parameters with same key
- Fix app settings table will not load local settings in Run Panel
- Set import app settings dialog always top

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java [hunk 1]
```diff
@@ -30,17 +30,29 @@
...
 public class AppSettingModel implements TableModel {
 
     private static final String[] TITLE = {"Key", "Value"};
+    private static final String FUNCTIONS_WORKER_RUNTIME_KEY = "FUNCTIONS_WORKER_RUNTIME";
+    private static final String AZURE_WEB_JOB_STORAGE_KEY = "AzureWebJobsStorage";
+    private static final List<Pair<String, String>> DEFAULT_APP_SETTINGS = Arrays.asList(
+            Pair.of(FUNCTIONS_WORKER_RUNTIME_KEY, "java"),
+            Pair.of(AZURE_WEB_JOB_STORAGE_KEY, "")
+    );
+    public static final String REQUIRED_APP_SETTINGS_PROMPTION = "%s is required parameter";
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java]: a required

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java [hunk 2]
```diff
@@ -63,6 +75,14 @@ public String getColumnName(int col) {
 
     @Override
     public boolean isCellEditable(int row, int col) {
+        if (!isRowValid(row)) {
+            return false;
+        }
+        final Pair<String, String> target = appSettings.get(row);
+        // Should not modify FUNCTIONS_WORKER_RUNTIME and AzureWebJobsStorage
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java]: return !(FUNCTIONS_WORKER_RUNTIME_KEY.equals(target.getKey()) || (AZURE_WEB_JOB_STORAGE_KEY.equals(target.getKey()) && col == 0))

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/ui/FunctionRunPanel.java [hunk 3]
```diff
@@ -108,7 +109,7 @@ public void disposeEditor() {
 
     @Override
     protected void resetFromConfig(@NotNull FunctionRunConfiguration configuration) {
-        if (!configuration.isFirstTimeCreated()) {
+        if (MapUtils.isNotEmpty(configuration.getAppSettings())) {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/ui/FunctionRunPanel.java]: Will this line makes FUNCTIONS_WORKER_RUNTIME_KEY not readable from local.settings.json?

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/ui/FunctionRunPanel.java]: No, `local.settings.json` will be loaded in `createUIComponents()` by `appSettingsTable.loadLocalSetting();`.

When `resetFromConfig` was invokerd by IntelliJ, if appSettings in configuration is empty, which means this is a new configuration, we will use local settings and if configuration is not empty, which means users used to save configuration before, and we will users configuration instead

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java]: Fixed

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java (hunk 2 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/component/table/AppSettingModel.java]: Fixed

**Pull request merged**